pkg/designate/dbsync.go

```diff
@@ -78,10 +80,10 @@ func DbSyncJob(
 	initContainerDetails := APIDetails{
 		ContainerImage:       instance.Spec.DesignateAPI.ContainerImage,
 		DatabaseHost:         instance.Status.DatabaseHostname,
-		DatabaseUser:         instance.Spec.DatabaseUser,
```

Comment:
OK this likely has to change because username and secret name are now dynamic, they change.   The job hash here cannot be determined with username/secret name.   Heat controller has the same problem.   somehow other contrllers like keystone etc. do not need to pass DB username/secret here and i am hoping this does not imply some huge change to kolla config or something